host/src/main/java/org/openjdk/skara/host/HostUserDetails.java

```diff
@@ -75,7 +73,6 @@ public String fullName() {
         if (name != null) {
             return name;
         }
```

Comment:
Nice improvement, but `fullName` can now be as simple as:
```java
public String fullName() {
    if (name == null) {
        name = nameSupplier.get();
    }
    return name;
}
```

Reply:
Sure, can change it